js/api/Surveys.js

```diff
@@ -141,19 +151,19 @@ export function cacheParseSurveys(survey) {
     realm.write(() => {
       realm.create('Survey', {
         id: survey.id,
-        active: survey.get('active') ? true : false,
+        active: survey.get('active') ? true : false, // eslint-disable-line no-unneeded-ternary
```

Comment:
Why not [simplify](http://eslint.org/docs/rules/no-unneeded-ternary) this expression vs. disabling the rule?